docs/release-process.md

```diff
@@ -0,0 +1,54 @@
+# Release Process
+
+When releasing code, we take the view that the `master` branch is always deployable. Whenever
+we merge a branch into `master`, this is automatically deployed into our staging environment
+(known as `development` in the DfE Cloud Platform).
+
+When deploying to production, however, we take the following steps.
+
+## 1. Update the Changelog and create a pull request
+
+* Create a branch from develop for the release called `release-x` where `x` is the release number
+* Move all features from the `Pending` section of `[CHANGELOG.md](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/CHANGELOG.md)` to a new heading with the release
+number linked to a diff of the two latest versions, together with the date in the following format:
+
+  ```markdown
+  ## [Release X] - 2019-01-01
+
+  ...
+
+  [Release X]: https://github.com/DFE-Digital/dfe-teachers-payment-service/compare/release-x..previous-release
+  ```
+
+* Document the changes in the commit message as well
+* Push the branch
+* Create a tag for the release in the format `release-x`
```

Comment:
Let's not do this until the PR is accepted and merged to prevent having to get tags deleted on everyone's machines that have fetched if we decide to change the release commit.